### PR TITLE
Adding Performance Tests

### PR DESCRIPTION
**Pull request by maneeshaheshan**

### CODE REVIEW

File: performance-tests/apps/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.9.2</version>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <microservice.mainClass>org.apim.lrtest.Application</microservice.mainClass>
+    </properties>
+
+</project>
```
Comment (chaminda) [inline performance-tests/apps/pom.xml]: New line character missing here.

File: performance-tests/apps/pom.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (chaminda) [inline performance-tests/apps/pom.xml]: copy rights are old

File: performance-tests/apps/dependency-reduced-pom.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chaminda) [inline performance-tests/apps/dependency-reduced-pom.xml]: This is a temporary file and shouldn't be in the repo

File: performance-tests/apps/pom.xml (hunk 1 above)
Comment (maneeshaheshan) [reply performance-tests/apps/pom.xml]: Requested changes are committed.

**Comment by chaminda** [comment]
closing since aged and might break existing functionalities, please send a new PR if this is applicable.

**Pull request closed without merging**